playground/nextjs/pages/survey.tsx

```diff
@@ -45,6 +46,28 @@ export default function SurveyForm() {
                     <h1>hello world</h1>
                 </div>
             </div>
+
+            {/* Add spacer to push the bottom button down */}
+            <div style={{ height: '70vh', display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+                <p>Scroll down to see the bottom feedback button</p>
+            </div>
+
+            {/* Bottom feedback button */}
+            <div style={{ padding: '20px', textAlign: 'center', borderTop: '1px solid #eee' }}>
+                <button
+                    className="survey-feedback-button"
+                    style={{
+                        padding: '10px 20px',
+                        background: '#1D4AFF',
+                        color: 'white',
+                        borderRadius: '4px',
+                        border: 'none',
+                        cursor: 'pointer',
+                    }}
+                >
+                    Feedback (Bottom)
+                </button>
+            </div>
```

Comment:
no need to pay attention to this code really - it's just a button I added to the nextjs playground